src/OrbitGl/AccessibleTimeline.h

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_ACCESSIBLE_TIMELINE_H_
+#define ORBIT_GL_ACCESSIBLE_TIMELINE_H_
+
+#include "AccessibleCaptureViewElement.h"
+
+class TimelineUi;
```

Comment:
You should be able to `#include "TimelineUi` here instead of the forward declaration, we want to avoid forward declarations if possible.

Reply:
Alternatively, you could also just pass a `CaptureViewElement*` to the constructor instead of `TimelineUi` since you're not using any of the functionality `TimelineUi` provides, and get rid of this completely.